value/reflectcache_test.go

```diff
@@ -69,6 +70,8 @@ func TestToUnstructured(t *testing.T) {
 		{Data: `{"a":1}`, Expected: map[string]interface{}{"a": int64(1)}},
 		{Data: `0`, Expected: int64(0)},
 		{Data: `0.0`, Expected: float64(0)},
+		{Data: `{}}`, ExpectedErrorMessage: "error decoding object from json: unexpected trailing data after object end"},
```

Comment:
It is unfortunate for a case like `{ "a": 1, "b": {}}, "c": 3}` that this error message does not indicate where in input string the error was found, since that could help the author of the JSON find and correct the mistake easier.  Can we use `InputOffset` somehow to improve the error?